test/fixtures/append.css

```diff
@@ -1,3 +1,4 @@
+@custom-media --a (--a);
```

Comment:
Oh, you are right. It's been awhile and I have no idea why this file exists. Let me double check.